src/makeCalculator.js

```diff
@@ -1,10 +1,39 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  const resultObj = {
+    result: 0,
+
+    add(number) {
+      resultObj.result += number;
+    },
+
+    subtract(number) {
+      resultObj.result -= number;
+    },
+
+    multiply(number) {
+      resultObj.result *= number;
+    },
+
+    divide(number) {
+      resultObj.result /= number;
+    },
+
+    operate(callback, number) {
+      callback(number);
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - When defining methods within an object, you should use the method shorthand instead of the `function` keyword. Replace `operate: function(callback, number) {` with `operate(callback, number) {` to comply with the checklist item 3.